src/config/dark-sites.config

```diff
@@ -163,6 +163,7 @@ carl.gg
 cascadr.co
 catvibers.me
 cdkeys.com
+cdn.minlor.net/$
```

Comment:
Why the `$`?

Reply:
I got a little bit confused sorry about that. removed it now